dd-sdk-android/src/main/kotlin/com/datadog/android/rum/tracking/ViewAttributesProvider.kt

```diff
@@ -0,0 +1,18 @@
+package com.datadog.android.rum.tracking
+
+import android.view.View
+
+/**
+ * Provides the extra attributes for the  as Map<String,Any?>.
+ */
+interface ViewAttributesProvider {
+
+    /**
+     * Add extra attributes to the default attributes Map.
+     * @param view the [View].
+     * @param attributes the default attributes Map. Usually this contains some default
+     * attributes which are determined and added by the SDK. Please make sure you do not
+     * override any of these. (e.g. target.classname, target.resourceId)
```

Comment:
If they're not meant to override SDK attributes, we need to provide a full list of those attributes, not just examples. 
Maybe redirect them to the `RumAttributes` object where all the standard values are defined ?

Reply:
I think I will do this. Actually this reminds me that I need to extract those internal attributes from here to RumAttributes